Windows/XinputDevice.cpp

```diff
@@ -68,6 +70,21 @@ static int LoadXInputDLL() {
 		return -1;
 	}
 
+	/* 100 is the ordinal for _XInputSetStateEx, which returns the same struct as XinputSetState, but with extra data in wButtons for the guide button, we think...
+   Let's try the name first, though - then fall back to ordinal, then to a non-Ex version (xinput9_1_0.dll doesn't have Ex) */
+	PPSSPP_XInputSetState = (XInputSetState_t)GetProcAddress((HMODULE)s_pXInputDLL, "XInputSetStateEx");
```

Comment:
In this case, we can just go for XInputSetState, I think there's no Ex and we don't need to worry about ordinals.

That said, I dunno if it's supported everywhere (I think it is), we could potentially nullcheck when setting the vibration rather than failing here with -1 if it doesn't load.

-[Unknown]

Reply:
I think Ex also exists, at least I saw references to it in other software through google search.